server/weather.go

```diff
@@ -12,6 +12,8 @@ import (
 	"time"
 )
 
+//fake comment
```

Comment:
Looks real to me.

Reply:
@Fingel Git's "Blame" feature has Matt listed on this comment. I guess he was trying to test out how comments work in Go. Not sure how this slipped by me